google/cloud/storage/client.h

```diff
@@ -184,7 +193,11 @@ class Client {
    *     Valid types for this operation include `PredefinedAcl`,
    *     `PredefinedDefaultObjectAcl`, `Projection`, and `UserProject`.
    *
-   * @throw std::runtime_error if the operation fails.
+   * @throw std::runtime_error if there is a permanent failure, or if there more
+   *     transient failures than allowed by the current retry policy.
+   *
+   * @par Idempotency
+   * This operation is always idempotent, it fails if the bucket already exists.
```

Comment:
Nit: The comma here should be a semicolon or period instead.

...here and in `CreateBucketForProject`.

Reply:
Done.